dasharo-stability/capsule-update.robot

```diff
@@ -140,19 +203,20 @@ Get BIOS Version
     RETURN    ${bios_version}
 
 Upload Required Files
+    Log To Console    PREPARE: Upload Files
+
     ${file_name}=    Get File Name Without Extension    ${CAPSULE_FW_FILE}
 
     Check If Capsule File Exists    ./dl-cache/edk2/${file_name}_max_fw_ver.cap
     Check If Capsule File Exists    ./dl-cache/edk2/${file_name}_wrong_cert.cap
     Check If Capsule File Exists    ./dl-cache/edk2/${file_name}_invalid_guid.cap
 
+    Set DUT Response Timeout    5m
     Power On
     Boot System Or From Connected Disk    ubuntu
-
     IF    '${DUT_CONNECTION_METHOD}' == 'pikvm'
         Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
```

Comment:
Does it have to be global? `Go To Ubuntu Prompt` uses suite variable with the same name, and according to https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#variable-scopes only test variables can't be used in suite setup. If suite variable works here, please change this set of keywords to `Go To Ubuntu Prompt`, if not, please provide a log with failure.

Reply:
@krystian-hebel changed in: 3417ae2f8e267a5ad154a2654a4a5c84b563ed22